utils/utils.go

```diff
@@ -17,9 +17,18 @@ import (
 	"github.com/pborman/uuid"
 )
 
-const testXattr = "trusted.glusterfs.test"
-const volumeIDXattr = "trusted.glusterfs.volume-id"
-const gfidXattr = "trusted.gfid"
+const (
+	testXattr     = "trusted.glusterfs.test"
+	volumeIDXattr = "trusted.glusterfs.volume-id"
+	gfidXattr     = "trusted.gfid"
+)
+
+var (
+	PathMax     = unix.PathMax
```

Comment:
Do you need to expose these functions to other packages?  If not, you probably want them using lower case letters.